### ISSUE TITLE
Color contrast ratio of keyboard focus present on "C#" control is 1.277:1 which is less than 3:1: A11y_Azure Function_Create new project_Non-text Contrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-azurefunctions#4482]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Nov2024; #DesktopApp; #A11yMAS; #WCAG1.4.11; #Non-text Contrast; #AILimited; #Win11; #FTP; #External; #External:VSCode; #FirstParty:Bug246460;

### Environment and OS details:
Application name: Azure Tools for VS Code
URL: Visual studio code

### Reproduction Steps:

1. Open Visual studio code
2. TAB till extension button and press ENTER then search for "Azure Functions"
3. TAB till install and press ENTER.
4. TAB till "Azure" button and press ENTER.
5. TAB till "Azure function" control and hit ENTER.
6. TAB till "Create new project" control and hit ENTER.
7. Open accessibility insights for windows and check color contrast ratio of keyboard focus on "C#".
8. Observe the issue whether Color contrast ratio of keyboard focus present on "C#" control is less than 3:1 or not.

### Actual behavior:
Color contrast ratio of keyboard focus present on "C#" control is 1.277:1 which is less than 3:1

### Expected behavior:
Color contrast ratio of keyboard focus present on "C#" control should be greater than or equals to 3:1

### User Impact:
Low vision users will not be able to understand where the keyboard focus is present if color contrast ratio will be less than 3:1.

### Attachment:

![Image](https://github.com/user-attachments/assets/ac7f18c5-838e-4639-9995-4300ef171b73)

**Comment by jinglouMSFT** [comment]
Not sure what theme you are using, but mine has the 21:1 ratio:

![Image](https://github.com/user-attachments/assets/10616e55-8fab-4c3f-8ed2-fc7d8a3cb023)

**Comment by merriemcgaw** [comment]
@jinglouMSFT The contrast ratio requirement is for non-text contrast ratios, rather than the text contrast ration. So in this case they're testing that the dark blue focus indication color is not different enough from the black background color. This may need to go to Office Hours, as I don't know if a decision has been made universally for this UI since the white text/blue background also indicates focus and meets the contrast ratio requirement. @scottaohara might be able to answer us here, but Office Hours may be quicker.

**Comment by jinglouMSFT** [comment]
VSCode has resolved the linked bug, which we automatically inherit the fix.  However, I am not sure if the fix really fixed the reported issue. If I am not mistaken, based on the proposed fix in the linked bug, the design was to increase the contrast between the text and the background. The reported bug is the contrast between the background and the keyboard focus border, which is still under 3:1 (In the dark theme. It's 2.846 : 1 as I ran the test today).  @kupatkar99, can you please help verify if the external bug (linked above) actually resolved the issue you reported?  If yes, then this bug should be also resolved and closed.  Thanks.

**Comment by Yash14j** [comment]
Issue is not reproducing hence closing the bug

#closed;

**Comment by jinglouMSFT** [comment]
Closing per above input.
